test/integration/targets/rpm_key/tasks/rpm_key.yaml

```diff
@@ -187,6 +187,33 @@
       - result is success
       - result is not changed
 
+- name: Issue 83493 - Verify key fingerprints. This should fail due to multiple key signatures
+  rpm_key:
+    key: https://ansible-ci-files.s3.amazonaws.com/test/integration/targets/rpm_key/linux_signing_key.pub
+    fingerprint: 1B09 0CE4 7562 66D7 1203 13F6 3BAC 309A 6EB4 25D5
```

Comment:
Correct me if I'm wrong, but isn't this a breaking change?

I think the current case is that users would only need to list one of the keys which appears in the import, and this test suggests that *all* keys would need to be listed for verification. I can imagine scenarios where users upload all their signing keys to a single file and then check for different keys in different modules. This change would break that usage if I have how it currently works right.